Makefile

```diff
@@ -4,15 +4,18 @@ TAG = v1.2.0
 PREFIX = gcr.io/google_containers
 FLAGS = 
 
+VERSION = 1.2.0
```

Comment:
We have now TAG and VERSION which are supposed to be the same. Could you please use just one of them?